i18n/de.toml

```diff
@@ -326,9 +326,9 @@ auto = "Automatisch"
 label = "Sprache"
 
 [settings.sponsorToken]
-expires = "Dein Sponsortoken läuft {inXDays} ab. {getNewToken} und aktualisiere deine Konfigurationsdatei."
+expires = "Dein Sponsor-Token läuft {inXDays} ab. {getNewToken} und aktualisiere deine Konfigurationsdatei."
```

Comment:
Warum? Was ist an Sponsortoken falsch? Diese Bindestricheritis finde ich grenzwertig- gibts dafür eine anzuwendende Regel?

Reply:
@andig Falsch ist „Sponsortoken“ nicht, aber möglicherweise nicht so leicht lesbar, weil es ein eher unübliches Wort ist. Laut Duden _kann_ man „in unübersichtlichen Zusammensetzungen“ einen Bindestrich setzen, man muss aber nicht: https://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich#D22
Meine Anregung sollte diesbezüglich helfen (und nicht nerven).

Reply:
Unüblich ja, unübersichtlich ist da für mein Empfinden aber nichts. Schön wäre wenn wir es überall einheitlich machen würden.

Reply:
Hier würd ich jetzt erstmal bei "Sponsortoken" bleiben. Vor allem weil wir das eigentlich konsistent in den anderen Systemen (Docs, Generierung, ...) auch so gemacht haben.